lib/spark_api/models/message.rb

```diff
@@ -13,11 +14,17 @@ def save(arguments={})
         end
         false
       end
+
       def save!(arguments={})
         results = connection.post self.class.path, {"Messages" => [ attributes ]}, arguments
         true
       end
-      
+
+      def replies(args = {})
+        arguments = {:_expand => "Body, Sender"}.merge(args)
+        @replies ||= Message.collect(connection.get("#{self.class.path}/#{self.Id}/replies", arguments))
```

Comment:
That is a really good question. Looking at this line, I would expect new replies would not show up, however I've been working with this all week and new replies are in fact showing up. I realized it's because I'm creating a new message object every time I refresh the replies:
```
def refresh
    @replies = Message.find(params['message_id']).replies(:_limit => 25).reverse
    render layout: false
end
```
The refresh method is only called with ajax, and a `.js.erb` template then updates the view. While it doesn't matter right now, it's probably best to remove the `@replies ||= `. That could cause someone some confusion in the future if `@message.replies` were ever used.